cli/cmd/inject.go

```diff
@@ -522,3 +534,32 @@ func overwriteRegistry(image, newRegistry string) string {
 	}
 	return registry + imageName
 }
+
+func throwInjectError(report *inject.Report) []error {
+
+	errs := []error{}
+
+	const (
+		disabledAutomountServiceAccountToken = "disabled_automount_service_account_token_account"
+		hostNetworkEnabled                   = "host_network_enabled"
+		sidecarExists                        = "sidecar_already_exists"
+		udpPortsEnabled                      = "udp_ports_enabled"
```

Comment:
Given this function is all about `inject.Report`, I think it's better to move it into `report.go` and make it a method of that struct. That will also avoid having to repeat these consts.

Reply:
@alpeb Done! You can find that refactor in [this](https://github.com/linkerd/linkerd2/pull/4346/commits/69ac7fc0b0cdd6ed2688a8ac343d03ce6f23a3f2) commit. (Had to rebase against master because some of the tests didn't seem to run)